docs/src/test-components-js.md

```diff
@@ -195,93 +195,168 @@ npm run test-ct
 
 Refer to [Playwright config](./test-configuration.md) for configuring your project.
 
-## Test stories
+## API reference
```

Comment:
I'd put this under "Stories" and "Under the hood" to make sure users learn the basics before diving into details of the API.